accounts/tests.py

```diff
@@ -1,30 +1,208 @@
-# from django.test import TestCase
-
-
-# class TestSignupView(TestCase):
-#     def test_success_get(self):
-
-#     def test_success_post(self):
-
-#     def test_failure_post_with_empty_form(self):
-
-#     def test_failure_post_with_empty_username(self):
-
-#     def test_failure_post_with_empty_email(self):
-
-#     def test_failure_post_with_empty_password(self):
-
-#     def test_failure_post_with_duplicated_user(self):
-
-#     def test_failure_post_with_invalid_email(self):
-
-#     def test_failure_post_with_too_short_password(self):
-
-#     def test_failure_post_with_password_similar_to_username(self):
-
-#     def test_failure_post_with_only_numbers_password(self):
-
-#     def test_failure_post_with_mismatch_password(self):
+from django.contrib.auth import SESSION_KEY, get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+User = get_user_model()
+
+
+class TestSignUpView(TestCase):

```

Comment:
```suggestion
class TestSignupView(TestCase):
```
uについて、
もともと小文字のため、小文字のままにしておきましょう。
この最終課題では前置詞は小文字のままのようです